README.md

```diff
@@ -1,594 +1,65 @@
-Nylas Node.js SDK  [![Travis build status](https://travis-ci.org/nylas/nylas-nodejs.svg?branch=master)](https://travis-ci.org/nylas/nylas-nodejs)
-======================
+# Nylas Node.js SDK  
 
-Installation
-------------
-Install the Nylas SDK:
+[![Travis build status](https://travis-ci.org/nylas/nylas-nodejs.svg?branch=master)](https://travis-ci.org/nylas/nylas-nodejs)
 
-`npm install nylas` or `yarn add nylas`
+This is the GitHub repository for the Nylas Node SDK and this repo is primarily for anyone who wants to make contributions to the SDK or install it from source. If you are looking to use Node to access the Nylas Email, Calendar, or Contacts API you should refer to our official [Node SDK Quickstart Guide](https://docs.nylas.com/docs/quickstart-node).
 
+The Nylas Communications Platform provides REST APIs for [Email](https://docs.nylas.com/docs/quickstart-email), [Calendar](https://docs.nylas.com/docs/quickstart-calendar), and [Contacts](https://docs.nylas.com/docs/quickstart-contacts), and the Node SDK is the quickest way to build your integration using JavaScript.
 
-API Overview
-------------
+Here are some resources to help you get started:
 
-Every resource (i.e., messages, events, contacts) is accessed via an instance of `Nylas`. Before making any requests, be sure to call `config` and initialize the `Nylas` instance with your `clientId` and `clientSecret`. Then, call `with` and pass it your `accessToken`. The `accessToken` allows `Nylas` to make requests for a given account's resources.
+- [Nylas SDK Tutorials](https://docs.nylas.com/docs/tutorials)
```

Comment:
np: let's use the Node anchor link https://docs.nylas.com/docs/tutorials#section-node-js